docker/pg_tests/scripts/tests/several_delta_backups_test.sh

```diff
@@ -55,4 +56,3 @@ psql -f scripts/amcheck.sql -v "ON_ERROR_STOP=1" postgres
 
 diff /tmp/dump1 /tmp/dump2
 /tmp/scripts/drop_pg.sh
-rm ${TMP_CONFIG}
```

Comment:
What is the reason to remove this?

Reply:
Probably a weird mixup.
Editing the file added a newline at end of file, and I accidentally took that for me adding this.
I didn't recall why, so I removed it from my commit. Sorry...